openpype/hosts/resolve/api/lib.py

```diff
@@ -278,6 +279,12 @@ def create_timeline_item(media_pool_item: object,
             clip_data.update({"endFrame": source_end})
 
         print(clip_data)
+
+        if timeline_in:
```

Comment:
Should this be?
```suggestion
        if timeline_in is not None:
```
Or does default load always start at timeline at 0 anyway?